xrpl/utils/txn_parser/utils/nodes.py

```diff
@@ -0,0 +1,56 @@
+"""Helper functions to normalize an affected node."""
+
+from typing import List, Union, cast
+
+from typing_extensions import Literal, TypedDict
+
+from xrpl.models import CreatedNode, DeletedNode, ModifiedNode, TransactionMetadata
+from xrpl.utils.txn_parser.utils.types import BalanceType
+
+
+class Fields(TypedDict):
+    """Model for possible fields."""
+
+    Account: str
+    Balance: Union[BalanceType, str]
+    LowLimit: BalanceType
+    HighLimit: BalanceType
+
+
+class NormalizedNode(TypedDict):
```

Comment:
Some of these fields need to be optional, since not all of the node types have them.